tests/test_move.py

```diff
@@ -60,3 +88,88 @@ def test_move_dir(self):
         if self.preserve_time:
             dst_file2_info = dst_fs.getinfo("bar/baz.txt", namespaces)
             self.assertEqual(dst_file2_info.modified, src_file2_info.modified)
+
+
+class TestMove(unittest.TestCase):
+    def test_move_file_tempfs(self):
+        with open_fs("temp://") as a, open_fs("temp://") as b:
+            dir_a = a.makedir("dir")
+            dir_b = b.makedir("subdir")
+            dir_b.writetext("file.txt", "Content")
+            fs.move.move_file(dir_b, "file.txt", dir_a, "here.txt")
+            self.assertEqual(a.readtext("dir/here.txt"), "Content")
+            self.assertFalse(b.exists("subdir/file.txt"))
+
+    def test_move_file_fs_urls(self):
+        # create a temp dir to work on
+        with open_fs("temp://") as tmp:
+            path = tmp.getsyspath("/")
+            subdir_src = tmp.makedir("subdir_src")
+            subdir_src.writetext("file.txt", "Content")
+            tmp.makedir("subdir_dst")
+            fs.move.move_file(
+                "osfs://" + join(path, "subdir_src"),
+                "file.txt",
+                "osfs://" + join(path, "subdir_dst"),
+                "file.txt",
```

Comment:
I wonder if it's worth using a different filename on both sides (i.e. not `"file.txt"` both times), just as an additional check that the filename parameters aren't getting muddled up, or the files aren't being accidentally moved-but-not-renamed?
(same comment applies to several of the other tests too)